internal/pkg/cli/app_delete.go

```diff
@@ -383,7 +409,7 @@ func buildAppDeleteCommand() *cobra.Command {
 		}),
 	}
 
-	cmd.Flags().StringVarP(&vars.name, nameFlag, nameFlagShort, tryReadingAppName(), appFlagDescription)
+	cmd.Flags().StringVarP(&vars.name, nameFlag, nameFlagShort, "", appFlagDescription)
```

Comment:
I just looked at the code for `tryReadingAppName` - it returns an empty string when there is no workspace. Therefore, it seems to be fine keeping `tryReadingAppName` here? what do you think